README.md

```diff
@@ -29,6 +29,9 @@ composer require smi2/phpclickhouse
 
 ### OR install submodule
 
+`composer` require php 7.1, but phpClickHouse can work on php 5.5 without autoload and `composer`
```

Comment:
Just note this http://php.net/supported-versions.php Shouldn't encourage 5.5 here ;)